website/manual.html

```diff
@@ -0,0 +1,43 @@
+<!-- Copyright 2018-2019 the Deno authors. All rights reserved. MIT license. -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Deno Manual</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
```

Comment:
Please, let's not block users from scaling the website, especially critical on mobile devices in terms of accessibility.
[Never use maximum-scale=1.0](https://a11yproject.com/posts/never-use-maximum-scale/)
[Interfering with a user agent's ability to zoom](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html#failure)
So, let's remove `maximum-scale=1.0` and `user-scalable=0` from every page.

Reply:
I have already opened https://github.com/denoland/deno/pull/1734 to fix that on `index.html`, but please, let's not add it to the `manual.html`.
`all_benchmark.html` already has only `width=device-width, initial-scale=1.0` as expected.

Reply:
Done